docs/architecture/governance.md

```diff
@@ -1,50 +1,84 @@
 ---
-title: Governance
+title: Governance and self-amendment
 last_update:
-  date: 3 July 2023
+  date: 27 February 2024
 ---
 
-## What is Self Amendment?
+Tezos incorporates a built-in, on-chain mechanism for proposing, selecting, testing, and activating protocol upgrades without the need to hard fork.
+This mechanism makes Tezos a self-amending blockchain and allows any user to propose changes to the [economic protocol](./architecture#the-tezos-self-amending-protocol), which defines the possible blockchain operations and how they are processed.
```

Comment:
this link is wrong, sorry this is my fault, I think it should be (please check before applying):
```suggestion
This mechanism makes Tezos a self-amending blockchain and allows any user to propose changes to the [economic protocol](../architecture#the-tezos-self-amending-protocol), which defines the possible blockchain operations and how they are processed.
```

Reply:
Yup, thanks for catching that.